datadog/resource_datadog_synthetics_test_.go

```diff
@@ -278,7 +285,7 @@ func syntheticsTestRequestHeaders() *schema.Schema {
 		Description:  "Header name and value map.",
 		Type:         schema.TypeMap,
 		Optional:     true,
-		ValidateFunc: validators.ValidateHttpRequestHeader,
+		ValidateFunc: validators.ValidateHttpRequestHeader, // TODO we might need one of these for initialApplicationArguments
```

Comment:
No, the `ValidateHttpRequestHeader` is to check that the values are compatible with http (e.g. no invisible characters). There is no need for such checks in application arguments, I think.
```suggestion
		ValidateFunc: validators.ValidateHttpRequestHeader, // TODO we might need one of these for initialApplicationArguments
```